cpp/src/parquet/size_statistics_test.cc

```diff
@@ -376,4 +376,22 @@ TEST_F(SizeStatisticsRoundTripTest, LargePage) {
                                                         /*byte_array_bytes=*/{90000}}));
 }
 
+TEST_F(SizeStatisticsRoundTripTest, MaxLevelZero) {
```

Comment:
Should we add a file to parquet-testing that has the def level histogram not present for max_def==1?